CONTRIBUTING.md

```diff
@@ -1,18 +1,139 @@
-# Contributing
+# Contributing to ICU4X Project
+
+`ICU4X` is an open source project and welcomes everyone to participate.
+
+In order to provide meaningful contributions, it is important to familiarize yourself with a set of documents which describe the [structure](./docs/charter.md) and [style](./docs/style-guide.md) used by the project.
+
+The list of [open issues](triaging.md) represent the current focus points of the project, and the [help wanted](https://github.com/unicode-org/icu4x/issues?q=is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee+) lists the issues which we don't have resources to work on right now, but would consider accepting if someone volunteered to work on them.
```

Comment:
I think the link to triaging.md needs to point to the docs directory. However, that seems like an odd place to link to when the link text is "open issues".

Reply:
Can you recommend which way you'd like me to remodel this link? (title or url?)

Reply:
Leave this as-is and follow up with another PR after you merge this one.  #149